frontend/src/pages/Chat/ChatComponent.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { useState, useEffect } from "react";
+import OnlineUsersList from "./OnlineUsers";
+import ChatWindow from "./ChatWindow";
+import { getHistoryMessages } from "../../services/auth.service";
```

Comment:
you should use the `constants.ts` file to define an endpoint like `export const AUTH_HISTORY_MESSAGES = `${AUTH_URL}/messages`;
`
and in the backend file of `src/routes/message.route.ts`, write an endpoint like:
`router.get('/:id', messageController.getConversation);`